docs/ResultsList.md

```diff
@@ -0,0 +1,45 @@
+# Search Bar
+
+## Description
+
+`ResultsList` is a VTEX Component that represents the autocomplete from the search bar. This component can be imported and used by any VTEX App.
```

Comment:
I think it's better to refer to the interface name instead of the component name. The component name it's part of our public API. The component name is implementation details.